epsagon/trace.py

```diff
@@ -113,8 +205,8 @@ def set_timeout_handler(self, context):
             if original_timeout <= TIMEOUT_GRACE_TIME_MS:
                 return
 
-            modified_timeout = (
-                original_timeout - TIMEOUT_GRACE_TIME_MS) / 1000.0
+            modified_timeout = \
```

Comment:
Basically using '\' in Python is a bad practice (https://www.python.org/dev/peps/pep-0008/#maximum-line-length)

Reply:
K